src/nodes.coffee

```diff
@@ -2437,6 +2439,26 @@ class IcedRuntime extends Block
     if @isEmpty() then []
     else               super o
 
+  inlineRuntime: (lefthand) ->
+    if iced.text
+      # If the compiler is ran from the browser and it uses the inline
+      # runtime, the runtime will already have the 'text' property we
+      # can just "pass along".
+      runtime_str = iced.text
+    else
+      # If not (compiler is stand alone), use readFileSync to get the
+      # runtime.
+      fs = require('fs')
+      runtime_str = fs.readFileSync "extras/inline-runtime.js", 'utf-8'
```

Comment:
Oh, oops, this part is still not finished. And you have a solution for this too, but I misread your comment, sorry! I'll move the generation to `lib/` instead of `extras/` and fix this as well.